modules/jaggery-apps/portal/gadgets/consent_management/js/gadget.js

```diff
@@ -350,6 +385,29 @@ function addActions(container) {
     });
 }
 
+function showWarning(){
+    $("#message").html(warningDialog);
+    message({
+        title: "Consent Selection",
+        content: 'Please select all consents in order to proceed...',
+        type: 'warning',
+        okCallback: function () {
+        }
+    });
+}
+
+function showConfirm(){
+    $("#message").html(confirmationDialog);
+    message({
+        title: "Consent Confirmation",
+        content: 'Are you sure you want to update/revoke this consent? This is not reversible.',
```

Comment:
shall we make this "....this operation is not reversible."

Reply:
Fixed with 9418a6ddb9355ce7854912891df37e75a94e5682